### PR TITLE
MYR-54 : rocksdb.sys_vars fails due to missing include files

### PR DESCRIPTION
**Pull request by george-lorch**
- Copied in upstream files to new directory mysql-test/rocksdb.sys_vars/include
- Fixed up all rocksdb.sys_vars tests to use new locations

### CODE REVIEW

**Comment by george-lorch** [comment]
Upstream patch sent as https://github.com/facebook/mysql-5.6/pull/529

File: mysql-test/suite/rocksdb.sys_vars/include/correctboolvalue.inc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+##
```
Comment (laurynas-biveinis) [inline mysql-test/suite/rocksdb.sys_vars/include/correctboolvalue.inc:1]: Could use a better header comment ("maps TRUE FALSE 1 0 to 1 0")

Comment (laurynas-biveinis) [reply mysql-test/suite/rocksdb.sys_vars/include/correctboolvalue.inc:1]: Does not handle ON OFF?

**Review by laurynas-biveinis — APPROVED** [review]
Very minor comments

File: mysql-test/suite/rocksdb.sys_vars/include/correctboolvalue.inc (hunk 1 above)
Comment (george-lorch) [reply mysql-test/suite/rocksdb.sys_vars/include/correctboolvalue.inc:1]: Good ideas, probably I should re-do this to handle all common variants of on, ON, On, off, OFF, Off, true, TRUE, True, false, FALSE, and False and send it upstream.

**Comment by george-lorch** [comment]
OK, touched up correctboolvalue.inc, please take a peek

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**